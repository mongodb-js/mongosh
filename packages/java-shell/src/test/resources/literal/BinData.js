// command checkResultClass
new BinData(16, 'MTIzNA==')
// command checkResultClass
BinData(16, 'MTIzNA==')
// command checkResultClass
new BinData(4, 'MTIzNA==')