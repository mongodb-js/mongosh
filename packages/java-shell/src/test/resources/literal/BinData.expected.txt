BinaryResult: { "$binary" : "MTIzNA==" , "$type" : 16}
BinaryResult: { "$binary" : "MTIzNA==" , "$type" : 16}
BinaryResult: { "$binary" : "MTIzNA==" , "$type" : 4}